### ISSUE TITLE
Java 11 support

### ISSUE THREAD

**Issue by IanAshley42** [thymeleaf/thymeleaf-extras-eclipse-plugin#84]
I have installed the plugin into Eclipse using http://www.thymeleaf.org/eclipse-plugin-update-site/.

What I found is that it doesn't work (th: tags not auto-completed) when Eclipse is running under Java 11. If I change Eclipse to run under Java 9, by editing the eclipse.ini file, then it works a treat.

I am using Eclipse 2019-06 on a Mac. I had the same issue with Eclipse 2019-03.

Is there a way to make the plugin work with Java 11+?

**Comment by delfuego** [comment]
I just discovered this as well. Any thoughts on how to make this work?

**Comment by IanAshley42** [comment]
I got it to work by running Eclipse under an older JVM. When installing Eclipse (at least on my Mac) it prompts for the JVM to use. I selected 9.0.4 and it works fine.

If you already have Eclipse installed you can just edit the `eclipse.ini` file. Look for `-vm`

```
...
-vm
/Library/Java/JavaVirtualMachines/jdk-9.0.4.jdk/Contents/Home/bin
...
```

**Comment by ultraq** [comment]
Looks like one of the dependencies (JAXB) uses the javax.activation package, which isn't included in Java 9+.  I've fixed this in the 3.0 version currently under development - no ETA on release date though.

**Comment by cypher256** [comment]
Is there a planned release date for Eclipse plugin 3.0.0?
